apis/r/R/utils-single-cell-experiment.R

```diff
@@ -0,0 +1,78 @@
+.anndata_to_sce_reduced_dim <- function(x) {
```

Comment:
Since we'll likely add support for exporting to other bioc classes in the future I suggest we rename utils-single-cell-experiment.R to utils-bioc.R, as a common place for all bioconductor-related utilities.